php-ts-mode.el

```diff
@@ -76,10 +76,10 @@
     "elseif" "enddeclare" "endforeach" "endif" "endswitch"
     "endwhile" "enum" "extends" "final" "finally" "for" "foreach"
     "fn" "function" "global" "if" "implements" "include_once"
-    "include" "insteadof" "interface" "namespace" "never" "new"
+    "include" "insteadof" "interface" "namespace" "new"
     "private" "protected" "public" "readonly" "require_once" "require"
     "return" "static" "switch" "throw" "trait" "try" "use"
-    "while" "yield" "yield from")
```

Comment:
`yield` and `from` are declared as independent tokens and must be displayed in some other way. Let's work on it later as TODO.

https://github.com/tree-sitter/tree-sitter-php/blob/1a40581b7a899201d7c2b4684ee34490bc306bd6/grammar.js#L1395C1-L1398